testing-unit-ts/mocha/bucket_pair.ts

```diff
@@ -0,0 +1,23 @@
+import * as aws from "@pulumi/aws"
+import * as pulumi from "@pulumi/pulumi";
+
+export class BucketPair extends pulumi.ComponentResource {
+
+    contentBucket: aws.s3.Bucket;
+    logsBucket: aws.s3.Bucket;
+
+    constructor(contentBucketName: string, logsBucketName: string, opts: any) {
+        super("pulumi:examples:BucketPair", "BucketPair", {}, opts);
+
+        this.contentBucket = new aws.s3.Bucket("contentBucket", {
+            bucket: contentBucketName,
+        }, { parent: this });
+
+        this.logsBucket = new aws.s3.Bucket("logsBucket", {
+            bucket: logsBucketName,
+        }, { parent: this });
+
+        // Register output properties for this component
+        this.registerOutputs();
```

Comment:
For completeness, you should register the outputs like this:

```suggestion
        this.registerOutputs({
            contentBucket: this.contentBucket,
            logsBucket: this.logsBucket
        });
```

This is so Pulumi knows that everything has been done (I'll find the more technical term later on)

Reply:
(Will implement this change obviously, it's super-easy – big thanks for your review ;-)!)